tests/expressions_window/tests.py

```diff
@@ -722,6 +722,21 @@ def test_invalid_type_start_row_range(self):
                 frame=RowRange(start='a'),
             )))
 
+    @skipUnlessDBFeature('supports_over_clause')
+    def test_window_expression_within_subquery(self):
+        subquery_qs = Employee.objects.annotate(
+            highest=Window(FirstValue('id'), partition_by=F('department'), order_by=F('salary').desc())
+        ).values('highest')
+        highest_salary = Employee.objects.filter(pk__in=Subquery(subquery_qs))
```

Comment:
Do we need to use `Subquery` here? (If I remove it the test still functions — i.e. fails without patch, passes with it.)

Reply:
You're right! The `Subquery` is implicit in this case and not required.

```sql
SELECT
"expressions_window_employee"."id", "expressions_window_employee"."name", "expressions_window_employee"."salary", "expressions_window_employee"."department", "expressions_window_employee"."hire_date" 
FROM "expressions_window_employee" 
WHERE "expressions_window_employee"."id" IN (
    SELECT FIRST_VALUE(U0."id") OVER (PARTITION BY U0."department" ORDER BY U0."salary" DESC
) AS "highest" FROM "expressions_window_employee" U0)
```

Reply:
You're right! The `Subquery` is implicit in this case and not required.

```sql
SELECT
"expressions_window_employee"."id", "expressions_window_employee"."name", "expressions_window_employee"."salary", "expressions_window_employee"."department", "expressions_window_employee"."hire_date" 
FROM "expressions_window_employee" 
WHERE "expressions_window_employee"."id" IN (
    SELECT FIRST_VALUE(U0."id") OVER (PARTITION BY U0."department" ORDER BY U0."salary" DESC
) AS "highest" FROM "expressions_window_employee" U0)
```